middleware/geocodeJSON.js

```diff
@@ -11,7 +11,7 @@ var _ = require('lodash');
  * @param {string} [basePath]
  * @returns {middleware}
  */
-function setup(peliasConfig, basePath) {
+function setup(peliasConfig, basePath, labelGenerator) {
```

Comment:
I think that `labelGenerator` param is not needed (update corresponding unit test, too)